Makefile

```diff
@@ -83,27 +83,15 @@ test-bindings: bindings
 clean-bindings:
 	rm -rf $(PKG_BINDINGS_DIR)
 
-# Update nvml.h from the specied CUDA_VERSION development image
-update-nvml-h: CUDA_MAJOR := $(word 1,$(subst ., ,$(CUDA_VERSION)))
-update-nvml-h: CUDA_MINOR := $(word 2,$(subst ., ,$(CUDA_VERSION)))
-update-nvml-h: .copy-nvml-h
-	$(DOCKER) run \
-		--rm \
-		-v $(PWD):$(PWD) \
-		-w $(PWD) \
-		--user $$(id -u):$$(id -g) \
-		nvidia/cuda:$(CUDA_VERSION)-devel \
-			sed -i -E 's#[[:blank:]]+$$##g' $(GEN_BINDINGS_DIR)/nvml.h
-
-.copy-nvml-h:
-	if [[ $(CUDA_VERSION) == "" ]]; then echo "define CUDA_VERSION to update"; exit 1; fi
-	$(DOCKER) run \
-		--rm \
-		-v $(PWD):$(PWD) \
-		-w $(PWD) \
-		--user $$(id -u):$$(id -g) \
-		nvidia/cuda:$(CUDA_VERSION)-devel \
-			cp /usr/local/cuda-$(CUDA_MAJOR).$(CUDA_MINOR)/targets/x86_64-linux/include/nvml.h $(GEN_BINDINGS_DIR)
+# Update nvml.h from the Anaconda package repository
+update-nvml-h: CONDA_PACKAGE_FILE := $(shell wget -qO - https://conda.anaconda.org/nvidia/linux-64/repodata.json | grep -oh '"cuda-nvml-dev-[^"]*"' | tr -d '"' | sort -rV | head -n 1)
+update-nvml-h: NVML_VERSION := $(word 4,$(subst -, ,$(CONDA_PACKAGE_FILE)))
```

Comment:
One minor comment. From our perspective, we often want to pull in the `nvml.h` file associated with a particular CUDA release as this allows us to update the APIs gradually. This change means that we always pull in "HEAD". Is there a way to select a specific version or print available versions?

Reply:
Yes. We can pull the `repodata.json` with a specific label:

![Screenshot NVML](https://user-images.githubusercontent.com/16078332/143467691-143a8850-57d8-447a-8227-753d2b53ac82.png)

I will change for this.

Reply:
I add a variable `LABEL` for querying a specific version of `nvml.h`. The variable name `LABEL` can be changed if something else is preferred.

Reply:
Update: list all available packages and let the user pick one from the input:

```console
$ make update-nvml-h
Found 5 NVML packages:
  1 -- 11.5.50  -- nvidia/cuda-nvml-dev/11.5.50/linux-64/cuda-nvml-dev-11.5.50-h511b398_0.tar.bz2
  2 -- 11.4.120 -- nvidia/cuda-nvml-dev/11.4.120/linux-64/cuda-nvml-dev-11.4.120-hb8c74d6_0.tar.bz2
  3 -- 11.4.43  -- nvidia/cuda-nvml-dev/11.4.43/linux-64/cuda-nvml-dev-11.4.43-he36855d_0.tar.bz2
  4 -- 11.3.58  -- nvidia/cuda-nvml-dev/11.3.58/linux-64/cuda-nvml-dev-11.3.58-hc25e488_0.tar.bz2
  5 -- 11.3.58  -- nvidia/cuda-nvml-dev/11.3.58/linux-64/cuda-nvml-dev-11.3.58-h70090ce_0.tar.bz2

Pick a NVML package to update ([1]-5): 1

NVML version: 11.5.50
Package: nvidia/cuda-nvml-dev/11.5.50/linux-64/cuda-nvml-dev-11.5.50-h511b398_0.tar.bz2
File: cuda-nvml-dev-11.5.50-h511b398_0.tar.bz2
Download URL: https://api.anaconda.org/download/nvidia/cuda-nvml-dev/11.5.50/linux-64/cuda-nvml-dev-11.5.50-h511b398_0.tar.bz2

Updating nvml.h to 11.5.50 ...
Successfully Updated nvml.h to 11.5.50.
```

Reply:
This is a great improvement.
Rather than printing the path to the tarball though, I think it would be more useful to print the timestamp here so we can disambiguate which is the latest when there are two with the same version. Or better yet, only print the latest one if there are multiple ones with the same version.

Reply:
> I think it would be more useful to print the timestamp here so we can disambiguate which is the latest when there are two with the same version. Or better yet, only print the latest one if there are multiple ones with the same version.

@klueska I see. The anaconda API returns a JSON file that contains the information we need (version / platform / URL / upload time), but I think it could be hard to use the basic command-line tools only (`wget` / `grep` / `sort` / `tr`) to parse them. Can I assume the user always has a `python3` or `jq` executable to parsing the JSON response?